src/librustdoc/formats/item_type.rs

```diff
@@ -48,7 +48,9 @@ crate enum ItemType {
     ProcAttribute = 23,
     ProcDerive = 24,
     TraitAlias = 25,
-    Generic = 26,
+    TyAssocType = 26,
+    TyAssocConst = 27,
+    Generic = 28,
```

Comment:
Please don't change the existing values. It'll break if there is a documentation (with dependencies) generated with different rustdoc versions.

Reply:
I see, thanks. Will do. I put them above `Generic` because `"generic"` currently isn't present in the `itemTypes` array in `search.js`. and I thought that's why `Generic` has to be the last variant. Instead, I will add `"generic"` to the script.